src/streams/VirgilStreamCipherBase.ts

```diff
@@ -52,13 +52,13 @@ export class VirgilStreamCipherBase {
 	 * result in an error being thrown.
 	 * This method also automatically calls `dispose`.
 	 */
-	final () {
+	final (dispose: boolean = false) {
```

Comment:
Let's make this `true` by default, because otherwise it breaks backwards compatibility and will introduce memory leaks in existing codebases.